Add dashboard.stakeaptos.com and postman collection to community contributions

pakaplace opened the pull request:
### Description
Adding the validator dashboard (dashboard.stakeaptos.com ) and postman collection our Paymagic team made.

clay-aptos reviewed (changes requested):
Thanks for the contribution, Parker. Since this is external to Aptos.dev, it should actually land in this list (at the top as latest):
https://aptos.dev/community/external-resources

By editing:
https://github.com/aptos-labs/aptos-core/blob/main/developer-docs-site/docs/community/external-resources.md

Still, I see the following error when trying toggling from one network to another after loading an example:
`Error - Validator not found at that address. Check your selected nework and address`

Can we resolve that too? Thanks!

pakaplace commented:
Hi @clay-aptos , I moved it to external resources and I fixed the previewnet example bug. Our RPC endpoint was screwed up when you tested it, although the mainnet example never broke. It's ready for review, thanks.

clay-aptos commented:
I resolved the merge conflict and added Paymagic to the spellcheck list in:
https://github.com/aptos-labs/aptos-core/pull/7203

Asking @michelle-aptos and @saharct to review and approve that.
